src/components/nav/NavModal.js

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useState } from 'react';
+import './NavModal.scss';
+import SearchModal from './SearchModal';
+
+function NavModal({ colseModal }) {
+  const [inputValue, setInputValue] = useState('');
+  const [searchList, setSearchList] = useState([]);
+  const [onSearchModal, setOnSearchModal] = useState(false);
+  const [searchData, setSearchData] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/products.json')
+      .then(response => response.json())
+      .then(result => setSearchList(result));
+  }, []);
+
+  const searchInput = e => {
+    setInputValue(e.target.value);
+    setOnSearchModal(true);
+    searchList.filter(item => {
+      if (item.name.includes(inputValue) == true) {
+        setSearchData(item.name);
+      }
+    });
+  };
+
+  return (
+    <div>
+      <div className="search-Modal">
+        <div className="search-Main">
+          <input
+            type="text"
+            placeholder="따끈따근 신제품이 궁금하다면?"
+            value={inputValue}
+            onChange={searchInput}
+          />
+          <a href="#!">
+            <img
+              className="search-Btn"
+              src={process.env.PUBLIC_URL + '/images/Nav/search.png'}
+              alt="searchBtn"
+            />
+          </a>
+          <button onClick={colseModal}>x</button>
+        </div>
+        {onSearchModal === true ? <SearchModal testData={searchData} /> : null}
```

Comment:
다음과 같이 수정해볼 수 있을 것 같습니다.
```suggestion
        {onSearchModal && <SearchModal testData={searchData} />}
```

Reply:
testData라는 props 프로퍼티명이 최종적으로 머지될 때도 사용될 이름인가요?

Reply:
그리고 onSearchModal이라는 state은 boolean 데이터를 가지는 변수인데, boolean naming convention에 맞게 앞에 is를 붙이는 방식으로 수정해주세요!
`e.g. isSearchModal`